src/demandlib/vdi/regions.py

```diff
@@ -603,8 +700,8 @@ def get_load_curve_houses(self):
         for house in self.houses:
             t_limit = namedtuple("temperature_limit", "summer winter")
             tl = t_limit(
-                summer=house["summer_temperature_limit"],
-                winter=house["winter_temperature_limit"],
+                summer=house.get("summer_temperature_limit", 15),
+                winter=house.get("winter_temperature_limit", 5),
```

Comment:
Aren't these temperatures in the norm? If so, I think it's okay to set them as defaults.

Reply:
Correct, these are the values used per default in the norm.

Reply:
If the default value is part of the norm. I agree.